modules/hal_nordic/nrfs/backends/Kconfig

```diff
@@ -25,4 +25,11 @@ config NRFS_BACKEND_TX_MSG_QUEUE_SIZE
 	range 1 16
 	default 8
 
+config NRFS_BACKED_MAX_CLB_COUNT
+	int "Max supported callbacks"
+	range 1 16
+	default 4
+	help
+	  Maximal number of registered callbacks when connection is established.
```

Comment:
I think it would be better to allow an arbitrary number of callbacks by using a linked list. You can find examples in subsys/pm/policy.

Reply:
Shouldn't the number of callback be equal to the number of nrfs services? Each nrfs service should have one user.

Reply:
Changed to linked list.